packages/manager/core/shell-client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ovh-ux/manager-react-shell-client",
-  "version": "0.8.0-alpha.0",
+  "version": "1.0.0",
```

Comment:
**question:** Whats the need to manually modify the version?

Reply:
my pr is not ready yet

Reply:
You can mark is as Draft then.